chia/rpc/wallet_rpc_api.py

```diff
@@ -841,30 +842,30 @@ async def create_offer_for_ids(self, request):
             )
         if success:
             return {
-                "offer": trade_record.offer.hex(),
+                "offer": encode_offer_bytes(Offer.from_bytes(trade_record.offer).compress()),
```

Comment:
Maybe a function that turns an `Offer` into a bech32m representation of that offer (which would require a version number too).